Bump pitmp-maven-plugin to pitest 1.4.1

A week ago, pitest released a new 1.4.1 version (https://github.com/hcoles/pitest/releases/tag/pitest-parent-1.4.1) with a fix for https://github.com/hcoles/pitest/pull/499

When will pitmp upgrade the dependency and do a new release? Does it need anything more than bumping the pom.xml versions and the table on https://github.com/STAMP-project/pitmp-maven-plugin#releases ?

tried to simply bump the pitest version on the root pom, but it seems is not enough, and now some tests are failling:

```
------------------------------------------------------------
- running tests
------------------------------------------------------------
FAILED
-------- results
################ dhell
######## pom.xml.pitmp.noconf: FAILED
######## pom.xml.pitmp.conf1: FAILED
################ dnoo
######## pom.xml.pitmp.noconf: FAILED
######## pom.xml.pitmp.conf1: FAILED
################ dhell5
######## pom.xml.pitmp.noconf: FAILED
######## pom.xml.pitmp.conf1: FAILED
################ dnoo5
######## pom.xml.pitmp.noconf: FAILED
######## pom.xml.pitmp.conf1: FAILED
-------- diff
2,3c2,3
< ######## pom.xml.pitmp.noconf: FAILED
< ######## pom.xml.pitmp.conf1: FAILED
---
> ######## pom.xml.pitmp.noconf: OK
> ######## pom.xml.pitmp.conf1: OK
5,6c5,6
< ######## pom.xml.pitmp.noconf: FAILED
< ######## pom.xml.pitmp.conf1: FAILED
---
> ######## pom.xml.pitmp.noconf: OK
> ######## pom.xml.pitmp.conf1: OK
8,9c8,9
< ######## pom.xml.pitmp.noconf: FAILED
< ######## pom.xml.pitmp.conf1: FAILED
---
> ######## pom.xml.pitmp.noconf: OK
> ######## pom.xml.pitmp.conf1: OK
11,12c11,12
< ######## pom.xml.pitmp.noconf: FAILED
< ######## pom.xml.pitmp.conf1: FAILED
---
> ######## pom.xml.pitmp.noconf: OK
> ######## pom.xml.pitmp.conf1: OK
[ERROR] Command execution failed.
org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
    at org.apache.commons.exec.DefaultExecutor.executeInternal (DefaultExecutor.java:404)
    at org.apache.commons.exec.DefaultExecutor.execute (DefaultExecutor.java:166)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:804)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:751)
    at org.codehaus.mojo.exec.ExecMojo.execute (ExecMojo.java:313)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:208)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:290)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:194)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)
[INFO] ------------------------------------------------------------------------
```

This output is not very easy to dive deeper into it. Any clue you'd like to share @CaelInria ?

--- a/README.md
+++ b/README.md
@@ -215,11 +215,16 @@ If you want to check the number of Pseudo Tested Methods and/or Partially Tested
 ## Releases
 For PIT release...         | use PitMP release... | how to use PitMP
 -------------------------- | -------------------- | ----------------
+1.4.2                     | 1.3.5 | Maven Central
 1.4.0                      | 1.3.4, 1.3.3, 1.3.2, 1.3.1, 1.3.0, 1.2.0 | Maven Central
 1.3.2                      | 1.1.6, 1.1.5         | Maven Central
 1.3.1                      | 1.1.4                | Maven Central
 1.2.1, 1.2.2, 1.2.4, 1.2.5, 1.3.0  | not tested   |
 1.2.0, 1.2.3               | 1.0.1                | git clone & mvn install
+
+* pitmp-maven-plugin-1.3.5
+  - Dependency on Descartes v1.2.4 (hotfix for a regression in 1.3.3)
+  - Tested with PIT v1.4.2, Descartes v1.2.4, JUnit4 and JUnit5
 
 * pitmp-maven-plugin-1.3.4
   - Dependency on Descartes v1.2.4 (hotfix for a regression in 1.3.3)

--- a/pom.xml
+++ b/pom.xml
@@ -65,13 +65,13 @@
     <dependency>
       <groupId>org.pitest</groupId>
       <artifactId>pitest-maven</artifactId>
-      <version>1.4.0</version>
+      <version>1.4.2</version>
     </dependency>
 
     <dependency>
       <groupId>org.pitest</groupId>
       <artifactId>pitest-entry</artifactId>
-      <version>1.4.0</version>
+      <version>1.4.2</version>
     </dependency>
 
     <dependency>

